<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClippingWizardPageForm</class>
 <widget class="QWizardPage" name="ClippingWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Type</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_18">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="0">
           <widget class="QComboBox" name="m_strategyTypeComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QStackedWidget" name="m_stackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Extent</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_15">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_14">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_9">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_5">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label">
                         <property name="text">
                          <string>Lower Left X</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_llxLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="1" column="0">
                      <layout class="QGridLayout" name="gridLayout_6">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>Lower Left Y</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_llyLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="2" column="0">
                      <layout class="QGridLayout" name="gridLayout_7">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>Upper Right X</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_urxLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="3" column="0">
                      <layout class="QGridLayout" name="gridLayout_8">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>Upper Right Y</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_uryLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Dimension</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_23">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_16">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_12">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_19">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_5">
                         <property name="text">
                          <string>Initial Row</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_startRowLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="1" column="0">
                      <layout class="QGridLayout" name="gridLayout_20">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_6">
                         <property name="text">
                          <string>Initial Column</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_startColumnLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="2" column="0">
                      <layout class="QGridLayout" name="gridLayout_21">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_7">
                         <property name="text">
                          <string>Height</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_heightLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="3" column="0">
                      <layout class="QGridLayout" name="gridLayout_22">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_8">
                         <property name="text">
                          <string>Width</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_widthLineEdit">
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="title">
                 <string>Layer</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_25">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_24">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_13">
                     <item row="0" column="0">
                      <widget class="QComboBox" name="m_layerComboBox"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QRadioButton" name="m_aggregAllRadioButton">
                       <property name="text">
                        <string>Aggregate all geometries</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <layout class="QGridLayout" name="gridLayout">
                       <item row="0" column="0" colspan="2">
                        <widget class="QRadioButton" name="m_groupByRadioButton">
                         <property name="text">
                          <string>Group By Attribute</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <spacer name="horizontalSpacer">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeType">
                          <enum>QSizePolicy::Fixed</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item row="1" column="1">
                        <widget class="QComboBox" name="m_layerAttrComboBox">
                         <property name="enabled">
                          <bool>false</bool>
                         </property>
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="minimumSize">
                          <size>
                           <width>140</width>
                           <height>0</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="3" column="0">
                      <widget class="QRadioButton" name="m_selectedGeomRadioButton">
                       <property name="text">
                        <string>Use selecteds geometries</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <spacer name="verticalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_previewGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QFrame" name="m_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_strategyTypeComboBox</tabstop>
  <tabstop>m_llxLineEdit</tabstop>
  <tabstop>m_llyLineEdit</tabstop>
  <tabstop>m_urxLineEdit</tabstop>
  <tabstop>m_uryLineEdit</tabstop>
  <tabstop>m_startRowLineEdit</tabstop>
  <tabstop>m_startColumnLineEdit</tabstop>
  <tabstop>m_heightLineEdit</tabstop>
  <tabstop>m_widthLineEdit</tabstop>
  <tabstop>m_layerComboBox</tabstop>
  <tabstop>m_groupByRadioButton</tabstop>
  <tabstop>m_layerAttrComboBox</tabstop>
  <tabstop>m_selectedGeomRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_strategyTypeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>m_stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_groupByRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_layerAttrComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
